FarihaIS/VanRoomies
# feat(profile fragment): Integrate Frontend Profile Fragment and Implement Profile Bio Update

# Welcome to VanRoomies! 👋✈️

Fixes: #40
Fixes: #56 

## Description of the change:
Integration of the Profile Fragment with the backend and implementation of the bio update option.

## How to manually test:
1. Start up MongoDB
2. Run `npm run dev`
3. Go to Postman and run `POST Create New User`
4. Take the outputted `_id` and paste it into `ProfileFragment` (Declared at the top of the file).
5. Build and run the app
6. See if the information is correct and if the profile can be edited.
7. Switch fragments and come back. The description should persist!
